Source/AwsCommonRuntimeKit/crt/Checksums.swift

```diff
@@ -0,0 +1,29 @@
+//  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//  SPDX-License-Identifier: Apache-2.0.
+
+import struct Foundation.Data
+import AwsCChecksums
+
+extension Data {
```

Comment:
Why are these two checksums in `Checksums.swift` while the other two are in `Utilities.swift`?

Reply:
SHA1 and SHA256 are cryptographic hash functions, while CRC32 and CRC32C are checksum algorithms with hardware and software based implementation in C. SHA1 and SHA256 can be used as checksums, so I was a bit confused about their placement and initially put them in Checksums.swift. However, considering that they are conceptually different, belong to different modules in C, and have different interfaces, I kept them in Utilities.swift.

Reply:
Thanks, I have moved the three hash functions to a Hash.swift file. Now our Utilities.swift contains only internal utility functions, which is great.